plugins/module_utils/vmware.py

```diff
@@ -1127,6 +1127,10 @@ def get_vm(self):
                     vm_obj = vms[0]
         elif 'moid' in self.params and self.params['moid']:
             vm_obj = VmomiSupport.templateOf('VirtualMachine')(self.params['moid'], self.si._stub)
+            try:
+                _ = getattr(vm_obj, 'name')
```

Comment:
```suggestion
                getattr(vm_obj, 'name')
```

It looks like just `_` [fails the CI](https://0ecc18f904952e6b6a96-e5d1b2401c9f0fc1c62a4e20453ccdd0.ssl.cf5.rackcdn.com/867/e1919edb03a56dd4524fa2156e87d121452b194b/check/ansible-test-sanity-docker/90d5755/job-output.html#l1884). Maybe it's enough to just try to access the name. Otherwise, I think you need to use another variable like `tmpname` or `testname` or something.

Reply:
Thanks for your comment!
I removed `_`.

Reply:
I'm not sure about this, but maybe the linter marks this as an error because `_` is planned to be used as the wildcard for [pattern matching](https://www.python.org/dev/peps/pep-0636/) in a future Python version.